plugins/revealer/qt.py

```diff
@@ -0,0 +1,697 @@
+        logo.setAlignment(Qt.AlignCenter)
+        vbox.addSpacing(42)
+
+        self.load_noise = ScanQRTextEdit()
+        self.load_noise.setTabChangesFocus(True)
+        self.load_noise.textChanged.connect(self.on_edit)
+        self.load_noise.setMaximumHeight(33)
+
+        vbox.addWidget(WWLabel(_("<b>Enter your physical revealer code:<b>")))
+        vbox.addWidget(self.load_noise)
+        vbox.addSpacing(11)
+
+        self.next_button = QPushButton(_("Next"), self.d)
+        self.next_button.setDefault(True)
+        self.next_button.setEnabled(False)
+        vbox.addLayout(Buttons(self.next_button))
+        self.next_button.clicked.connect(self.d.close)
+        self.next_button.clicked.connect(partial(self.cypherseed_dialog, window))
+        vbox.addSpacing(21)
+
+        vbox.addWidget(WWLabel(_("or, alternatively: ")))
+        bcreate = QPushButton("Create a digital Revealer")
+        bcreate.clicked.connect(partial(self.make_digital, self.d))
+        bcreate.clicked.connect(partial(self.cypherseed_dialog, window))
+        vbox.addWidget(bcreate)
+        vbox.addSpacing(11)
+        vbox.addWidget(QLabel(''.join([_("<b>WARNING</b>: Printing a revealer and encrypted seed"), '<br/>',
+                                       _("on the same printer is not trustless towards the printer."), '<br/>',
+                                       ])))
+        vbox.addSpacing(11)
+        vbox.addLayout(Buttons(CloseButton(self.d)))
+
+        return bool(self.d.exec_())
+
+    def get_noise(self):
+        text = self.load_noise.text()
+        return ''.join(text.split()).lower()
+
+    def on_edit(self):
+        s = self.get_noise()
+        b = self.is_noise(s)
+        if b:
+            self.noise_seed = s[:-3]
+            self.user_input = True
+        self.next_button.setEnabled(b)
+
+    def code_hashid(self, txt):
+        x = to_bytes(txt, 'utf8')
+        hash = sha256(x).hexdigest()
+        return hash[-3:].upper()
+
+    def is_noise(self, txt):
+        if (len(txt) >= 34):
+            try:
+                int(txt, 16)
+            except:
+                self.user_input = False
+                return False
+            else:
+                id = self.code_hashid(txt[:-3])
+                if (txt[-3:].upper() == id.upper()):
+                    self.code_id = id
+                    self.user_input = True
+                    return True
+                else:
+                    return False
+        else:
+            self.user_input = False
+            return False
+
+    def make_digital(self, dialog):
+        self.make_rawnoise(True)
+        self.bdone(dialog)
+        self.d.close()
+        return
+
+    def bcrypt(self, dialog):
+        self.rawnoise = False
+        dialog.show_message(_(self.filename+" encrypted for Revealer "+ self.version + '_'+self.code_id+" saved as PNG and PDF at: <b>" + self.base_dir+self.filename+self.version + '_'+self.code_id+'</b>'))
```

Comment:
This will not work. Since the translation template won't contain the variables, it will never be translated this way. Better use

```python
_("{} encrypted for Revealer {}_{} saved as PNG and PDF at: <b>{}{}{}_{}</b>').format(self.filename, self.version, self.code_id, self.base_dir, self.filename, self.version, self.code_id)
```